public/assets/images/person.svg

```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="17" height="16" viewBox="0 0 17 16">
```

Comment:
let's create react components for this icon and put them in:
```src/components/icons/IconName/index.js```
and add an index.js inside icons folder that exports all icons like this:
```
import ChartIcon from 'Chart';
import PersonIcon from 'Person';

export {
    ChartIcon,
    PersonIcon,
};
```